it/src/main/resources/nakadi.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# colors
+OK="\033[32m"
+FAIL="\033[31m"
+RESET="\033[0m"
+
+GIT=`which git`
+DOCKER_COMPOSE=`which docker-compose`
+DIRECTORY=/tmp/nakadi
+REPO="https://github.com/zalando/nakadi"
+NAKADI_PORT="8080"
+NAKADI_ALIVE_TIMEOUT=240
+
+function validate {
+
+    [ -z "$DOCKER_IP" ] && {
+        echo -e "You need to set DOCKER_IP env variable ${FAIL}✗${RESET}"
+        exit 1;
+    }
+
+    [ "$DOCKER_COMPOSE" = "" ] && {
+        echo -e "You need to install docker-compose ${FAIL}✗${RESET}"
+        exit 1;
+    }
+
+    [ "$GIT" = "" ] && {
+        echo "Git can't be found in your system"
+        echo -ne "  ${OK}suggestion${RESET}: run '"
+        [ x$(uname) = "xLinux" ] && {
+            [ x$(which apt-get) != "x" ] && {
+                echo "apt-get install git' to install it"
+            } || {
+                echo "yum install git' to install it"
+            }
+        } || {
+            echo "brew install git' to install it"
+        }
+        echo ""
+        exit 1
+    }
+}
+
+function start_nakadi {
+    echo -n "About to clone Nakadi... "
+    rm -rf $DIRECTORY/
+    $GIT clone $REPO $DIRECTORY
```

Comment:
I think always starting the masert (or develop) without fixed version might make the test fragile. Or is it done by intention to see, if the API has changed?